using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,249 @@
+# Continuous Aggregates 
+TimescaleDB continuous aggregates allow users to speed aggregate queries which
+span large amounts of time by continuously calculating them in the background
+and materializing the aggregates in a separate view. Conntinuous aggregates use
+the background worker framework to run aggregates on a schedule as new data is
+inserted into a hypertable or when old data is modified. There's a bit more
+complexity to how they're implemented which you can read about in our [blog post
+announcing the feature][continuous-aggs-blog], but for now, suffice it to say
+that they can really help if you've been having trouble with long-running
+aggregate queries. 
+
+
+### Creating a Continuous Aggregate View [](create)
+Continuous aggregates are exposed to the user by extending the PostgreSQL
+[`CREATE VIEW`][postgres-createview] command using a `WITH` option, must contain
+a [`time_bucket`][time-bucket] function call in the group by clause and support
+all parallel-safe aggregates. 
```

Comment:
need more info on what are parallel-safe aggregates (point to postgres docs if they have a section?). Also, please give the list of supported parallel safe built in aggregates that we support.